### PR TITLE
Modify autoware_version not to use transient_local (workaround)

### PR DESCRIPTION
**Pull request by isamu-takagi**

### CODE REVIEW

File: system/autoware_version/src/autoware_version.cpp [hunk 1]
```diff
@@ -41,12 +42,21 @@ class AutowareVersionNode : public rclcpp::Node
     RCLCPP_INFO(get_logger(), "ROS_DISTRO=%s", message.ros_distro.c_str());
 
     publisher_ = create_publisher<autoware_system_msgs::msg::AutowareVersion>(
-      "autoware_version", rclcpp::QoS{1}.transient_local());
-    publisher_->publish(message);
+      "autoware_version", rclcpp::QoS{1});
+
+    auto callback = [this, message] { publisher_->publish(message); };
```
Comment (mitsudome-r) [inline system/autoware_version/src/autoware_version.cpp:47]: Could you add a `TODO` comment saying that this is just a temporary fix until rosbridge supports transient_local QoS.

**Review by mitsudome-r — APPROVED** [review]
LGTM

**Comment by mitsudome-r** [comment]
Merge this after CI is green

File: system/autoware_version/src/autoware_version.cpp (hunk 1 above)
Comment (isamu-takagi) [reply system/autoware_version/src/autoware_version.cpp:48]: Added.

**Review by mitsudome-r — APPROVED** [review]

**Pull request merged**